internal/service/advancedcluster/move_state_test.go

```diff
@@ -0,0 +1,79 @@
+package advancedcluster_test
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestAccAdvancedCluster_move_not_supported_sdkv2(t *testing.T) {
+	var (
+		projectID   = acc.ProjectIDExecution(t)
+		clusterName = acc.RandomClusterName()
+	)
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             acc.CheckDestroyCluster,
+		Steps: []resource.TestStep{
+			{
+				Config: configMoveFirst(projectID, clusterName),
+			},
+			{
+				Config:      configMoveSecond(projectID, clusterName),
+				ExpectError: regexp.MustCompile("Move Resource State Not Supported"),
+			},
+			{
```

Comment:
Why this final step?

Reply:
in some tests i got dangling resources not deleted, but it could be because something else, deleting these last steps, thanks